src/Menu/Menu.tsx

```diff
@@ -27,8 +27,10 @@ export type MenuDefaultProps = {
   isOpen: boolean;
   /** whether the menu should be closed when clicking outside */
   dismissOnClickOutside: boolean;
-  /** wheter the menu should be rendered on top or at the bottom of the trigger */
+  /** wheter the menu should be rendered on top or at the bottom of the trigger (bottom by default) */
   position: "top" | "bottom";
+  /** wheter the menu should be aligned with the start, the end or the center of the trigger (end by default) */
+  anchor: "start" | "center" | "end";
```

Comment:
could it make sense to take these props from the Popover props?
```
anchor: Popover.Props['anchor']
```

same for `position` above.

Reply:
Sure but `Popover.Props['position']` can also be "left" or "right", I don't know if we want to make it so general. 
Actually for the same reason i initially didn't include "center" as an option for `anchor`.

Reply:
👍 makes sense